Composer/ScriptHandler.php

```diff
@@ -453,4 +459,16 @@ protected static function useNewDirectoryStructure(array $options)
     {
         return isset($options['symfony-var-dir']) && is_dir($options['symfony-var-dir']);
     }
+
+    /**
+     * Returns true if the application bespoke autoloader is used.
+     *
+     * @param array $options Composer options
+     *
+     * @return bool
+     */
+    protected static function useSymfonyAutoloader(array $options)
+    {
+        return isset($options['symfony-app-dir']) && is_file($options['symfony-app-dir'].PATH_SEPARATOR.'autoload.php');
```

Comment:
Oh boy lots of goofs there for that few characters :D one - I meant `DIRECTORY_SEPARATOR` and two - is_file doesn't care if you just use `/`. Rebased and replaced with a `/`.